docs/source/overview/options.md

```diff
@@ -197,7 +197,7 @@ information of the evaluator contains following items:
                 "type": "accuracy",
                 "sub_types": [
                     {"name": "accuracy_score", "priority": 1, "goal": {"type": "max-degradation", "value": 0.01}},
-                    {"name": "f1_score"},
+                    {"name": "f1_score", "metric_config": {"task": "binary"}},
```

Comment:
it seems the old version doesn't have the parameter

Reply:
![image](https://github.com/microsoft/Olive/assets/13343117/edd2a9b0-317f-497f-b9b6-599d0b28c984)
Oh, yes. there is no `task` in v0.8.0 of torhmetircs. Do you know why we limit the version range as `>=0.7.0, <0.10.1`"? Can we update it?

Or I can fix the bert bug with other configs like `multiclass=2`

Reply:
Updated.

Reply:
Updated

Reply:
the version is pinned by Jambey, he has more context.

Reply:
Ic, anyhow I updated it with another parameter which exited in `>=0.7.0, <0.10.1`.

Reply:
We pinned the version some time last year when the new version that came out was incompatible with our evaluation code. 
there was a big change in the api. I think they introduced the tasks concept and it broke our code. 

I think we can revisit the versions if needed. Maybe check the torch vision version, map the config params and call the relevant torch metric functions for the version?

Reply:
Thanks. It is fine now. We change the logic for metric default config. Currently, we use inspect to generate the metric config with the origin default configs.  
Then no matter what kinda version user choose, just provide right parameters would be ok.